### DISCUSSION TITLE
When referring to an issue in projects beta - numbers don't match

### DISCUSSION THREAD

**Discussion by leighjboyd** [community/community#12709] [Projects and Issues]
I might call this a bug, or maybe its just a suggestion for improvement...

When I create a new issue in Projects beta, it gives a number in the left column.
This column seems to have little meaning, and can't be relied upon to act as any sort of reference number.
when typing in the project beta issue description, if you type #number it tries to link to the issue number but has no reference to the PROJECT BETA iSSUE NUMBER.

I find it frustrating that I must refer to the issue numbers, and that the issue numbers are not listed as a field in the project beta default fields 

to illustrate, in issues I have issue #15
<img width="760" alt="image" src="https://user-images.githubusercontent.com/99194378/157911465-722a6399-df9f-41a2-b700-4f40cba92793.png">

in projects the same issue is #20
<img width="953" alt="image" src="https://user-images.githubusercontent.com/99194378/157911612-e30af9c3-7c07-4f5a-8565-36df275a3831.png">

Suggestion
it makes me want a new field called issue number in the projects beta fields.

**Comment by nordhagen** [comment]
I was about to report the same thing. The number column on the far left is not useful and can be confusing.

My suggestion is to replace it with the real issue number to make it easier to find specific issues. Draft issues could just have a blank value here.

**Reply by mreekie** [reply]
The issue number is tied to the repository that the issue was created in.
I know that there's a lot of people who use multiple repositories with projects and maybe there's something in the works to make some sort of global issue ID?

It doesn't matter to those of us eager to move our teams to the new projects beta.
**We need issue numbers.**  It's how everyone on the team references issues.

We've had to use other tools such as Asana and jumped through hoops to add GitHub-like Task Ids to our tasks there.

**Comment by mjvandermeulen** [comment]
I was about to report the same thing as well.

I suggest that the issue # gets its own column in the home tab list as well. This column could be placed between the _Title_ and the _Assignnees_, with a '#', to avoid confusion.
